frontend/src/components/dashboard/dashboard.jsx

```diff
@@ -49,11 +51,21 @@ let Dashboard = React.createClass({
 
         <ul>
           <Item id='0' name='+ New Project' click={this.open} isCreateProject='true'/>
-          {
-            this.state.projects.map((project) => {
-              return <Item key={project.id} id={project.id} name={project.name} click={this.enterProject} delete={this.deleteProject} />;
-            })
-          }
+          <ReactCSSTransitionGroup transitionName='project'>
```

Comment:
This automatically applies css classes `project-enter`, `project-enter-active`, `project-leave` and `project-leave-active` for child components as they are added/removed.